cla-backend-go/project/service.go

```diff
@@ -61,6 +64,19 @@ func (s service) GetCLAGroupByID(projectID string) (*models.Project, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	log.Warnf("Checking for foundationSFID: %s CLA Groups", project.FoundationSFID)
+	pcgs, pcgErr := s.projectCGRepo.GetProjectsIdsForFoundation(project.FoundationSFID)
+	if pcgErr != nil {
+		return nil, pcgErr
+	}
+
+	if signedAtFoundationLevel(pcgs) {
+		project.FoundationLevelCLA = true
+	}
```

Comment:
Default is false then, right?

Reply:
Yes default is false. Either project level or no CLA groups association